### ISSUE TITLE
Failed to install

### ISSUE THREAD

**Issue by IrenXu** [chlubba/catch22#15]
Hi, I failed to install it in Python, would you mind checking out what is the problem?
And how to use catch22? Is there a GUI or maybe a handbook for detailed method?
![image](https://user-images.githubusercontent.com/77282283/105357879-5b2d3c00-5bed-11eb-99ab-34e60fd19d4b.png)

**Comment by chlubba** [comment]
Hi @IrenXu,

thanks for you interest in the catch22 package. The error message says a higher version (>=14.0, meaning 2015) of Visual Studio is required. Could you download the latest Visual Studio Connunity Version 2019 (or 2015/2017), install it and try again? Thanks, hope it works then!

Best,
Carl

**Comment by IrenXu** [comment]
Hi,

Thanks for the advice, i have downloaded Visual Studio Community 2019 yet it is still not working I guess I have to download C++ Build Tools.

So it means I cannot directly install it in anaconda environment?

**Comment by chlubba** [comment]
Yes, you need a compiler. Please let me know if it works for you with the C++ Build Tools. I may need to update the readme. Thanks

**Comment by IrenXu** [comment]
Hi,
I have successfully installed, thanks very much!
But is there a detailed instruction about how to use it?

**Comment by IrenXu** [comment]
I mean, are there requirements for the input dataset? I have a quite large dataset (about 300 rows 400-500 columns, each row represents a timeseries and each column represents a time point). Is this dataset could be upload as input directly?

**Comment by DamianJLin** [comment]
Hi, I'm having the issue as well, but after installation of C++ with Visual Studio Build Tools, it remains unfixed. I have also tried installing Visual Studio 2019, and the problem remains.

**Comment by sniperlucian** [comment]
here too - but looks like it expects mingw32 !?
